cpp/src/transform/jit/core/launcher.cpp

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "../core/launcher.h"
+#include <jit/parser.h>
+#include <jit/types_h_jit.h>
+#include <chrono>
+#include <cstdint>
+#include "../code/code.h"
+
+namespace cudf {
+namespace transformation {
+namespace jit {
+
+constexpr char prog_name[] = "unary_transform";
+
+const std::vector<std::string> Launcher::compilerFlags{"-std=c++14"};
+const std::vector<std::string> Launcher::headersName{cudf_types_h};
+
+/**
+ * @brief  Used to provide Jitify with strings that should be used as headers
+ *  during JIT compilation.
+ *
+ * @param filename  file which was requested to include in source
+ * @param stream    stream to pass string of the requested header to
+ * @return std::istream*
+ **/
+std::istream* headersCode(std::string filename, std::iostream& stream) {
+  return nullptr;
+}
+
+Launcher& Launcher::setProgram(std::string prog_file_name) {
```

Comment:
Is this dead code? why should this exist?

Reply:
@devavret I realize I have two set of `launcher.*` files for unary and binary ops. I will combine them.